R/stat-boxplot.r

```diff
@@ -45,6 +45,11 @@ StatBoxplot <- ggproto("StatBoxplot", Stat,
   {
     data <- remove_missing(data, na.rm, c("x", "y", "weight"), name="stat_boxplot",
       finite = TRUE)
+    if (!is.discrete(data$x) && data$group[1L] == NO_GROUP && any(data$x != data$x[1L])) {
```

Comment:
Did you check this? I think by this point factors have been converted into integers, so the appropriate test is `is.numeric(data$x)`.  And I think `all(data$group == NO_GROUP)` would be clearer